presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveFileFormats.java

```diff
@@ -291,16 +294,15 @@ public void testRcBinaryOptimizedWriter(int rowCount)
     public void testOrc(int rowCount, long fileSizePadding)
             throws Exception
     {
-        // Hive binary writers are broken for timestamps
-        List<TestColumn> testColumns = TEST_COLUMNS.stream()
-                .filter(TestHiveFileFormats::withoutTimestamps)
-                .collect(toImmutableList());
-
-        assertThatFileFormat(ORC)
-                .withColumns(testColumns)
-                .withRowsCount(rowCount)
-                .withFileSizePadding(fileSizePadding)
-                .isReadableByPageSource(new OrcPageSourceFactory(new OrcReaderOptions(), HDFS_ENVIRONMENT, STATS, UTC));
+        for (HiveTimestampPrecision timestampPrecision : List.of(MILLISECONDS, MICROSECONDS, NANOSECONDS)) {
```

Comment:
should we have this for other formats in this class?

Reply:
I'll add more.